### ISSUE TITLE
[Feature request]: Allow arguments to builtin `copy`

### ISSUE THREAD

**Issue by lmburns** [gokcehan/lf#1115]
I want to be able to use the builtin `copy` command to copy a given file's absolute path.

I've tried the following:

```zsh
cmd copy-realpath ${{
  local origin
  origin="$(realpath -- "$fx")"
  lf -remote "send $id copy '$origin'"
}}
```

However, when using the builtin `copy` command, it will either copy the current file selection or it will copy the file that is under the cursor. It would be much easier to allow an argument instead of having to clobber the `$XDG_DATA_HOME/lf/files` file to get this to work.

**Comment by Joyje** [comment]
I don't fully get what you are trying to do, but I believe this is what you want:

`cmd cpath %{{`
`    printf "Add file(s) to clipboard from path: "`
`    read cpath`
`    lf -remote "send $id toggle ${cpath}"`
`    lf -remote "send $id copy"`
`}}`

`map <some_key> cpath`

As for the second part of your post, why don't you just do this?

`map <some_key> cd /path/to/directory`

This doesn't seem to be an issue with lf though, so it should probably have been asked under discussions instead.

**Comment by lmburns** [comment]
@Joyje Yes that does work. However, allowing arguments to `mark-save`, etc. would be nice. I'll change the title of the issue
